superchain/superchain.go

```diff
@@ -129,7 +211,23 @@ var Addresses = map[uint64]*AddressList{}
 
 var GenesisSystemConfigs = map[uint64]*GenesisSystemConfig{}
 
+var Implementations = map[uint64]ContractImplementations{}
+
+// networkToChainID maps a network name to a chain ID. Only the superchain targets
+// are required to be in this mapping
+var networkToChainID = map[string]uint64{
```

Comment:
The L1 chain ID is already included in the superchain targets info (see `Superchains` global map)